### ISSUE TITLE
YOLOv5 Classification Success??

### ISSUE THREAD

**Issue by salihai** [ultralytics/yolov5#9499]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hey! How can i increase success of YOLOv5 classification model?
![ss](https://user-images.githubusercontent.com/54337470/191072662-014acd8b-5430-49df-8d53-ae5f14c8d52c.PNG)

**Comment by glenn-jocher** [comment]
@salihai hmm, you might want to try different augmentation strategies here (make sure albumentations is installed)
https://github.com/ultralytics/yolov5/blob/0b724c5b851b32bb3a8fbfab3cc2d68f93b4661e/utils/augmentations.py#L306-L339

or you could also try loading different torchvision architectures like resnet, efficientnet, etc., i.e.:
```
python classify/train.py --model efficientnet_b2
```

**Comment by salihai** [comment]
Okey thank you. I'll try and inform you about the results.

**Comment by frabob2017** [comment]
How to use this function?   I thought by doing data augmentation, you only change .yaml file 
[yolov5/data/hyps/hyp.scratch-low.yaml](https://github.com/ultralytics/yolov5/blob/11640698977724daf7982c9da398c2ee2f2b6e91/data/hyps/hyp.scratch-low.yaml#L25)

**Comment by salihai** [comment]
@frabob2017 classify_albumentations function is in yolov5/utils/augmentations.py file. You can use it by changing the parameters from here.

**Comment by frabob2017** [comment]
> @frabob2017 classify_albumentations function is in yolov5/utils/augmentations.py file. You can use it by changing the parameters from here.

it is a great learning platform for Yolo algorithm.

**Comment by salihai** [comment]
@glenn-jocher Hello again! There is one more thing I wonder about the question I asked earlier. During my summer internship, I was asked to perform a Yolo classification operation using the publicly available CheXpert dataset. I tried different augmentation strategies and different torchvision architectures with your suggestion here. But I couldn't find any solution that increased my accuracy. I also preprocessed the data in different ways, but it still didn't work. Does anyone have any work or knowledge of this dataset? How can I perform YOLOv5 classification on this data? Also what is the difference between top1_acc and top5_acc?
